src/main/java/com/igot/cb/config/RedisConfig.java

```diff
@@ -21,10 +22,17 @@ public class RedisConfig {
     @Value("${spring.redis.port}")
     private int redisPort;
 
+    @Value("${spring.redis.database:1}")
+    private int redisDatabase;
+
     @Bean
     @Primary
     public ReactiveRedisConnectionFactory reactiveRedisConnectionFactory() {
-        return new LettuceConnectionFactory(redisHost, redisPort);
+        RedisStandaloneConfiguration config = new RedisStandaloneConfiguration();
+        config.setHostName(redisHost);
+        config.setPort(redisPort);
+        config.setDatabase(redisDatabase);
```

Comment:
Do not set the database here... 
If you have to read any information from other index in future - it will complicate things.

Do make sure index is passed to methods which calls APIs in JedisPool object